### PR TITLE
Update import component to make use of updated env

### PR DESCRIPTION
**Pull request by novaturient95**
Env vuln fixes par of PR: #3298 
Validation run: [link](https://ml.azure.com/runs/faef2622-f97f-45bf-a414-fe169ff42632?wsid=/subscriptions/75703df0-38f9-4e2e-8328-45f6fc810286/resourceGroups/rg-sasumai/providers/Microsoft.MachineLearningServices/workspaces/sasum-westus3-ws&tid=7f292395-a08f-4cc0-b3d0-a400b023b0d2#)

### CODE REVIEW

File: assets/common/components/register_model/spec.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ display_name: Register model
 description:
   Register a model to a workspace or a registry. The component works on compute with [MSI](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-manage-compute-instance?tabs=python) attached.
 
-environment: azureml://registries/azureml/environments/python-sdk-v2/versions/20
+environment: azureml://registries/azureml-staging/environments/python-sdk-v2/versions/21
```
Comment (sarthaks95) [inline assets/common/components/register_model/spec.yaml:12]: Is this change accidental?

File: assets/training/model_management/components/validation_trigger_import/spec.yaml [hunk 2]
```diff
@@ -180,7 +180,7 @@ outputs:
 
 is_deterministic: True
 
-environment: azureml://registries/azureml/environments/python-sdk-v2/versions/20
+environment: azureml://registries/azureml/environments/python-sdk-v2/versions/22
```
Comment (sarthaks95) [inline assets/training/model_management/components/validation_trigger_import/spec.yaml:183]: Should this be 21 or 22?

**Review by sarthaks95 — CHANGES_REQUESTED** [review]

File: assets/common/components/register_model/spec.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 name: register_model
-version: 0.0.16
+version: 0.0.18
```
Comment (sarthaks95) [inline assets/common/components/register_model/spec.yaml:3]: 17 or 18?

**Review by sarthaks95 — CHANGES_REQUESTED** [review]

File: assets/common/components/register_model/spec.yaml (hunk 1 above)
Comment (novaturient95) [reply assets/common/components/register_model/spec.yaml]: Thanks for noticing this Sarthak. Updated now

File: assets/common/components/register_model/spec.yaml (hunk 3 above)
Comment (novaturient95) [reply assets/common/components/register_model/spec.yaml]: updated

File: assets/training/model_management/components/validation_trigger_import/spec.yaml (hunk 2 above)
Comment (novaturient95) [reply assets/training/model_management/components/validation_trigger_import/spec.yaml]: updated

**Review by sarthaks95 — APPROVED** [review]

**Pull request merged**